backend/uclapi/dashboard/tasks.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+import keen
+
+from celery import shared_task
+
+
+@shared_task
+def test_task(param):
+    return 'The test task executed with argument "%s" ' % param
+
+
+@shared_task
+def keen_add_event_task(title, data):
+    try:
+        keen.add_event(title, data)
+    except Exception:

```

Comment:
explicit errors should be here. @wilhelmklopp has already mentioned it. There should be a possible list of exceptions in keep docs.